src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -152,23 +152,23 @@ public function setAllowOverwrite($allow)
     }
 
     /**
-     * Sets the closures used for normalization.
+     * Sets the callbacks used for normalization.
      *
-     * @param \Closure[] $closures An array of Closures used for normalization
+     * @param callable[] $callbacks An array of callbacks used for normalization
      */
-    public function setNormalizationClosures(array $closures)
+    public function setNormalizationCallbacks(array $callbacks)
```

Comment:
@Tobion the typehint changes are BC breaks only for people extending the classes, because of the E_STRICT (and for most places, they probably don't need to extend it). The method renaming is the only BC break affecting the usage of the class.